src/spec-node/featuresCLI/info.ts

```diff
@@ -0,0 +1,85 @@
+import path from 'path';
+import { Argv } from 'yargs';
+import { CLIHost } from '../../spec-common/cliHost';
+import { getFeatureRef, getPublishedVersions } from '../../spec-configuration/containerFeaturesOCI';
+import { Log, LogLevel, mapLogLevel } from '../../spec-utils/log';
+import { createLog } from '../devContainers';
+import { UnpackArgv } from '../devContainersSpecCLI';
+import { getPackageConfig } from '../utils';
+
+export function featuresInfoOptions(y: Argv) {
+	return y
+		.options({
+			'log-level': { choices: ['info' as 'info', 'debug' as 'debug', 'trace' as 'trace'], default: 'info' as 'info', description: 'Log level.' },
+			'json': { type: 'boolean', default: false, description: 'Output in JSON format' },
```

Comment:
`up` is using `--log-format json`. Should we use `--output-format json` here? (`--log-format json` might make sense here too, we can add it later if needed.)